apps/src/clientApi.js

```diff
@@ -1,38 +1,71 @@
-/* global dashboard */
 // TODO(dave): Merge with the client API in /shared.
 // TODO: The client API should be instantiated with the channel ID, instead of grabbing it from the `dashboard.project` global.
+import queryString from 'query-string';
 
-module.exports = {
-  animations: clientApi('animations'),
-  assets: clientApi('assets'),
-  sources: clientApi('sources')
-};
+const _collectionType = Symbol('collectionType');
```

Comment:
nit: not sure if our linter enforces this, but I think we discourage leading underscores: https://github.com/airbnb/javascript#naming--leading-underscore . I'd call this `COLLECTION_TYPE`